addons/l10n_za/data/template/account.account-za.csv

```diff
@@ -1,112 +1,130 @@
+"213101","213101","Deposits received","liability_current","False"
+"213201","213201","Directors Loan Account - XXXXX","liability_current","False"
+"213202","213202","Directors Loan Account - XXXXX","liability_current","False"
+"310000","310000","Opening Balance and VAT Adjustments","equity","False"
+"320000","320000","Retained Income","equity","False"
+"330000","330000","Owners Equity","equity","False"
+"340000","340000","Retained Income (Opening Balance)","equity","False"
+"360000","360000","Profit and Loss (This Year)","equity_unaffected","False"
+"411001","411001","Product Sales 1","income","False"
+"411002","411002","Product Sales 2","income","False"
+"412001","412001","Service Sales 1","income","False"
+"412002","412002","Service Sales 2","income","False"
+"413001","413001","Grants Received","income","False"
+"414001","414001","Cost Recovery Transport","income","False"
+"421001","421001","Interest Earned","income","False"
+"422001","422001","Insurance payouts received","income","False"
+"422002","422002","Other Non Trade Revenue","income","False"
+"422003","422003","Discount received","income","False"
+"423001","423001","Bad Debts Recovered","income","False"
+"424001","424001","Profit or Loss from Asset Disposal","income","False"
+"424002","424002","Forex Exchange gain or loss","income","False"
+"424003","424003","Cash Difference Gain or Loss","income","False"
+"510001","510001","Materials Purchased 1","expense_direct_cost","False"
+"520001","520001","Services purchased 1","expense_direct_cost","False"
+"530001","530001","Other Cost of Sales","expense_direct_cost","False"
+"611001","611001","Accounting and Bookkeeping","expense","False"
+"611002","611002","Audit & Legal fees","expense","False"
+"611003","611003","Corp Gov & Secretarial Services","expense","False"
+"612001","612001","Asset & Repairs <R10000","expense","False"
+"612002","612002","Office Furniture & Equipment","expense","False"
+"612003","612003","Small Tools","expense","False"
+"612004","612004","Leasehold Improvement","expense","False"
+"612005","612005","Facilities, Infrastructure & Maintenance","expense","False"
+"612006","612006","Depreciation","expense","False"
+"613001","613001","Bank Charges","expense","False"
+"613002","613002","Interest Paid","expense","False"
+"613004","613004","Bad Debts","expense","False"
+"614001","614001","Staff Salaries","expense","False"
+"614002","614002","Directors Salaries","expense","False"
+"614003","614003","Cost of Recruitment","expense","False"
+"614004","614004","Provision for cash cost of leave","expense","False"
+"614005","614005","Provision for salary increases & bonuses","expense","False"
+"614006","614006","Consulting - Health & Safety","expense","False"
+"614007","614007","Sars - Paye","expense","False"
+"614008","614008","Sars - Uif","expense","False"
+"614009","614009","Sars - Sdl","expense","False"
+"614010","614010","Coida","expense","False"
+"614011","614011","SHEQ","expense","False"
```

Comment:
This doesn't make sense to me - I don't recognise these accounts, since there is no pre-defined template in ZA,
these seem to be company specific
The excel sheet seems to indicate that only the highlighted accounts are missing. I don't think we need to change all the codes

Reply:
I just did a quick search and Coida is a kind of health insurance in SA and SHEQ is a compliance organism related to health and environement. We don't need to create specific accounts for those though as it seems company specific.